addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -175,12 +175,10 @@ const FormEditor = options.Class.extend({
         if (field.custom && !field.string) {
             params.field.string = field.name;
         }
-        if (field.type === "description") {
-            if (field.description) {
-                params.default_description = _t("Describe your field here.");
-            } else if (["email_cc", "email_to"].includes(field.name)) {
-                params.default_description = _t("Separate email addresses with a comma.");
-            }
+        if (field.description) {
+            params.default_description = _t("Describe your field here.");
+        } else if (["email_cc", "email_to"].includes(field.name)) {
```

Comment:
I am not sure to understand why a description is needed for the `email_to` field as it seems that it is an always hidden field.